Docs/Images/image.html

```diff
@@ -0,0 +1,190 @@
+                        Cards
+                    </a>
+
+                    <a href="../Headers/header.html" class="tr-btn tr-btn-icon-label tr-btn-transparent">
+                        Headers
+                    </a>
+
+                    <a href="#" class="tr-btn tr-btn-icon-label tr-btn-transparent">
+                        Image
+                    </a>
+                    <a href="../input/input.html" class="tr-btn tr-btn-icon-label tr-btn-transparent">
+                        Input
+                    </a>
+                    <a href="../Text-Utilities/text-utilities.html" class="tr-btn tr-btn-icon-label tr-btn-transparent">
+                        Text Utilities
+                    </a>
+                    <a href="../Lists/lists.html" class="tr-btn tr-btn-icon-label tr-btn-transparent">
+                        Lists
+                    </a>
+                    <a href="../Modals/modal.html" class="tr-btn tr-btn-icon-label tr-btn-transparent">
+                        Modals
+                    </a>
+                    <a href="../Ratings/rating.html" class="tr-btn tr-btn-icon-label tr-btn-transparent">
+                        Ratings
+                    </a>
+                    <a href="../Snackbars/snackbar.html" class="tr-btn tr-btn-icon-label tr-btn-transparent">
+                        Snackbars
+                    </a>
+                    <a href="../Grid/grids.html" class="tr-btn tr-btn-icon-label tr-btn-transparent">
+                        Grid layouts
+                    </a>
+                </ul>
+
+            </div>
+        </aside>
+    </div>
+
+    <header class="tr-header main-header d-flex f-wrap">
+        <button data-target="sidebarMain" class="tr-btn tr-btn-icon hamburger"><i class="fas fa-bars"></i></button>
+        <div class="tr-heading">
+            <a href="../../index.html">
+                <h2 class="header-brand">
+                    <img class="logo" src="../../assets/tarang-favicon-2.png" alt="Tarang Logo">
+                    <span>arang</span> <span>UI</span>
+                </h2>
+            </a>
+        </div>
+        <nav class="tr-nav d-flex">
+            <a href="#" class="tr-btn tr-btn-icon-label tr-btn-link">
+                Docs
+                <i class="fas fa-file"></i>
+            </a>
+            <a href="https://github.com/kjais1720/Tarang-UI" target="_blank" rel="noopener noreferrer"
+                class="tr-btn tr-btn-icon-label tr-btn-link">
+                Github
+                <i class="fab fa-github"></i>
+            </a>
+        </nav>
+    </header>
+
+    <main class="main-content d-flex">
+        <div class="component-desc flex-col">
+            <h2>Images</h2>
+            <p>
+                Images are integral part of any website, they are really important to attract user's attention or give them some information, so handling images and making them responsible becomes really important.
```

Comment:
is that meant to be "responsible" or is it "responsive"..?

Reply:
Responsive, thanks for pointing it out, I have fixed it in the latest commit